src/assets/i18n/cs.json5

```diff
@@ -667,6 +667,26 @@
   "clarin.license.agreement.token.info": "Obdržíte e-mail s pokyny ke stažení.",
   // "clarin.license.agreement.signer.ip.address": "Ip Address",
   "clarin.license.agreement.signer.ip.address": "IP adresa",
+  // "clarin.license.agreement.signer.SEND_TOKEN": "Email",
+  "clarin.license.agreement.signer.SEND_TOKEN": "Email",
+  // "clarin.license.agreement.signer.NAME": "Name (given name and surname)",
+  "clarin.license.agreement.signer.NAME": "Jméno a příjmení",
+  // "clarin.license.agreement.signer.DOB": "Date of birth",
+  "clarin.license.agreement.signer.DOB": "Datum narození",
+  // "clarin.license.agreement.signer.ADDRESS": "Address",
+  "clarin.license.agreement.signer.ADDRESS": "Adresa",
+  // "clarin.license.agreement.signer.COUNTRY": "Country",
+  "clarin.license.agreement.signer.COUNTRY": "Země",
+  // "clarin.license.agreement.signer.EXTRA_EMAIL": "E-mail",
+  "clarin.license.agreement.signer.EXTRA_EMAIL": "E-mail",
+  // "clarin.license.agreement.signer.ORGANIZATION": "Organization",
+  "clarin.license.agreement.signer.ORGANIZATION": "Organizace",
+  // "clarin.license.agreement.signer.REQUIRED_ORGANIZATION": "Organization",
+  "clarin.license.agreement.signer.REQUIRED_ORGANIZATION": "Organizace",
+  // "clarin.license.agreement.signer.INTENDED_USE": "Intended use",
+  "clarin.license.agreement.signer.INTENDED_USE": "Zamýšlené použití",
+  // "clarin.license.agreement.signer.ACA_ORG_NAME_AND_SEAT": "Name and seat (address) of your academic institution",
+  "clarin.license.agreement.signer.ACA_ORG_NAME_AND_SEAT": "Název a sídlo vědecké instituce",
```

Comment:
akademicke

Reply:
I just copied it from the UFAL: https://github.com/ufal/clarin-dspace/blob/clarin-dev/dspace/modules/xmlui/src/main/webapp/i18n/messages_cs.xml#L1972